### ISSUE TITLE
time.Time comparison produces non-helpful diff

### ISSUE THREAD

**Issue by AlekSi** [stretchr/testify#989]
https://github.com/stretchr/testify/pull/895 broke diffs with time.Time:

In 1.5.1:
``` diff
-    Created: (xxx.Time) 2020-01-14T06:23:58Z,
-    Modified: (xxx.Time) 2020-01-14T06:23:58Z,
+    Created: (xxx.Time) 2018-08-05T09:30:53Z,
+    Modified: (xxx.Time) 2019-04-11T04:28:50Z,
```

In 1.6.1:
```diff
@@ -13,3 +13,3 @@
        wall: (uint64) 0,
-      ext: (int64) 63714579838,
+      ext: (int64) 63669058253,
        loc: (*time.Location)(<nil>)
@@ -20,3 +20,3 @@
        wall: (uint64) 0,
-      ext: (int64) 63714579838,
+      ext: (int64) 63690553730,
        loc: (*time.Location)(<nil>)
```

**Comment by Jozmen** [comment]
Can we expect a minor release with #895, before 1.7.0?

**Comment by boyan-soubachov** [comment]
> Can we expect a minor release with #895, before 1.7.0?

`go get github.com/stretchr/testify@master` will get you the latest version on `master` :)

**Comment by AlekSi** [comment]
#895 was released in 1.6.0 and was the source of this problem, not solution

**Comment by Jozmen** [comment]
sorry, I meant fix #895.

**Comment by HaraldNordgren** [comment]
@AlekSi @Jozmen @boyan-soubachov Fix proposed here if you want to take a look :) https://github.com/stretchr/testify/pull/1072
